cgi-bin/LJ/Talk.pm

```diff
@@ -2153,8 +2153,9 @@ sub icon_dropdown {
 
         # userpic browse button
         if ( $remote && $remote->can_use_userpic_select ) {
-            $ret .= '<input type="button" id="lj_userpicselect" value="Browse" />';
-            $ret .= LJ::Talk::js_iconbrowser_button();
+            my $metatext = $remote->iconbrowser_metatext ? "true" : "false";
+            my $smallicons = $remote->iconbrowser_smallicons ? "true" : "false";
+            $ret .= '<input type="button" id="lj_userpicselect" value="Browse" data-iconbrowser-metatext="' . $metatext . '" data-iconbrowser-smallicons="' . $smallicons . '"/>';
```

Comment:
nit: Use `qq{...}` to avoid having to concatenate quoted strings with mixed quotes and variables. Something like this:

```perl
$ret .= qq{<input type="button" id="lj_userpicselect" value="Browse" data-iconbrowser-metatext="$metatext" data-iconbrowser-smallicons="$smallicons"/>};
```

Reply:
good call, thanks!